cadc-util/build.gradle

```diff
@@ -16,15 +16,22 @@ sourceCompatibility = 1.8
 
 group = 'org.opencadc'
 
-version = '1.7.2'
+version = '1.7.3'
 
 description = 'OpenCADC core utility library'
 def git_url = 'https://github.com/opencadc/core'
 
 dependencies {
-    compile 'org.apache.logging.log4j:log4j-1.2-api:[2.17.2,3.0)'
-    compile 'org.apache.logging.log4j:log4j-api:[2.17.2,3.0)'
-    compile 'org.apache.logging.log4j:log4j-core:[2.17.2,3.0)'
+    //compile 'org.apache.logging.log4j:log4j-1.2-api:[2.17.2,3.0)'
```

Comment:
You've left the original (in the end correct?) compile statements in here - it'll be worth adding a comment about why with the line below. It might be obvious, but might not.

Reply:
I put the explanation with the currently active dependencies (hopefully temporary); it's implicit that it apllies to the commented out "normal" case because you can't have duplicates